### PR TITLE
find_untranslated.sh

### PR DESCRIPTION
**Pull request by ybznek**
Add simple script. It is like diff between directories, but it can compare 
**/bash/package_aide_installed.sh** vs **/ansible/package_aide_installed.yml**

Output from the script can be used like some checklisk

### CODE REVIEW

**Comment by mpreisler** [comment]
I have no idea what this is supposed to do.

File: shared/utils/find_untranslated.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+[ "$#" -lt 3 ] && {
+	echo "Usage:"
+	echo -e "\t$0 <original directory> <target directory> <path replacement ';' as separator>"
+
+	echo "Example:"
+	echo -e "\t$0 ./bash ./ansible '/(.*).sh;/\\1.yml'"
+	echo ""
+	echo "This file show 'diff' between directories & show not translated remediations"
```
Comment (jan-cerny) [inline shared/utils/find_untranslated.sh]: I suggest: This script compares two directories and looks for remediation scripts which are not implemented in one of the specified languages.

**Comment by redhatrises** [comment]
> I can't say I have a strong opinion but I wish something like this was integrated into the rest of our tools. Such as @iankko's SSG stats script. We can't just create new scripts every single time. In its current form I am against this being merged.

Agree with @mpreisler. Why not add to `find_untemplated.py`?

**Comment by ybznek** [comment]
I will close this. I don't have time to rewrite it in python right now.

**Pull request closed without merging**